packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -108,6 +134,9 @@ const scrollOverrides = (win, doc) => {
   // since we scroll down the page in takeScrollingScreenshots
   // and don't want the page size to change once we start
   // https://github.com/cypress-io/cypress/issues/6099
+
+  // Window.Event is private and also deprecated. See https://developer.mozilla.org/en-US/docs/Web/API/Window/event.
+  // @ts-ignore
   win.dispatchEvent(new win.Event('scroll'))
```

Comment:
I believe the link you referenced is for `window.event` not the `Event` interface.
```suggestion
  win.dispatchEvent(new Event('scroll'))
```
or you can add `Event` to: https://github.com/cypress-io/cypress/blob/90fe5027cd1b040a3c5ee9c881f92c437f90f2af/packages/driver/src/dom/elements/types.ts#L2

Reply:
whoops 😅. Updated in 5257d7e0b7 and also checked the tests for #6099 and looks good.